README.md

```diff
@@ -118,42 +118,42 @@ In the example below, the `inspector` variable will be used.  For the client-sid
 
 ### Validation
 
-* [type](#v_type)
```

Comment:
I'd rather keep the "v_" and "s_" prefixes since there is this pattern of having each thing work for both sanitization and validation. It will make more sense than using the word without the prefix (ex. using "type" instead of "v_type") for one of either sanitization or validation, and then having to choose something else for the other to disambiguate them.

Reply:
Once a heading is found on GH, they will call the first occurrence with the name and the rest with the suffix `-(occurrence)`. Something similar to: https://www.npmjs.com/package/slugger-unique. The hash will scroll the view to the element with the ID. Since there isn’t any element with the ID `v_type`, the browser will not find any element and thus ignore it. This is why the type link doesn’t work right now. However, `type` and `type-1` does work.

Reply:
Right, but I'd rather we fix it by renaming the anchor links to match the `v_type` and `s_type` pattern that the list of links is already using. I think it'll be more intuitive, easier to maintain long term.